mergJSON/mergJSONLibrary.livecodescript

```diff
@@ -0,0 +1,94 @@
+﻿script "ws.goulding.script-library.mergjson"
+/**
+
+Name: mergJSON Support Library
+
+Summary:
+Support functions for the mergJSON external
+
+Requires:
+
+- mergJSON
+
+*/
+
+on revLoadLibrary
+   if the target is not me then
+      pass revLoadLibrary
+   end if
+   
+   insert the script of me into back
+   
+   if the environment contains "development" then
+      set the _ideoverride of me to true
+   end if
+end revLoadLibrary
+
+on revUnloadLibrary
+   if the target is not me then
+      pass revUnloadLibrary
+   end if
+   
+   remove the script of me from back
+end revUnloadLibrary
+
+/**
+
+Parse JSON to a LiveCode array
+
+Parameters:
+pJSON (string): A UTF8 encoded JSON string
+
+Returns: A LiveCode array
+
+Description:
+JSON arrays are translated to LiveCode numerically indexed arrays 
+with keys 1...N and JSON objects are translated to LiveCode array
+key -> value pairs.
+
+*/
+
+function JSONToArray pJSON
+   local tArray,tKeys
+   repeat for each line tKey in mergJSONDecode(pJSON,"tArray")
+      put JSONToArray(tArray[tKey]) into tArray[tKey]
+   end repeat
+   
+   if tArray is an array then
+      return tArray
+   else
+      return textDecode(tArray,"UTF8")
+   end if
+end JSONToArray
+
+/**
+
+Parse a LiveCode array to JSON
+
+Parameters:
+pArray (array): A LiveCode array
+pForceRootType (optional enum): Provides the ability to force an element
+to be an object even if it appears to be a numerically indexed array or
+a string even if it appears to be a number.
```

Comment:
Sorry I missed this before -- I think you forgot to add the valid values of `pForceRootType` to this documentation comment.